File: JeffSebastian/playground/src/main/java/nl/uva/se/ast/expression/And.java
package nl.uva.se.ast.expression;

import nl.uva.se.visitor.Visitor;

public class And extends Binary {

	public And(int lineNumber, int offset, Expression left, Expression right) {
		super(lineNumber, offset, left, right);		
	}

	@Override
	public void accept(Visitor visitor) {
		visitor.visit(this);		
	}
	
}

File: FelipezConde/QuestionnaireLanguage/AST/Nodes/Expression/Binary/And.cs
using AST.Nodes.Interfaces;
using AST.Representation;

namespace AST.Nodes.Expression.Binary
{
    public class And : Binary, IExpression
    {
        public And(IExpression left, IExpression right, PositionInText position)
            : base(left, right, position)
        {}

        public override T Accept<T>(ASTVisitors.IVisitor<T> visitor)
        {
            return visitor.Visit(this);
        }

        public override string ToString()
        {
            return "&&";
        }
    }
}
